src/Actions/ManagesWorkflow.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Novu\SDK\Actions;
+
+use Novu\SDK\Resources\Workflow;
+
+trait ManagesWorkflow
+{
+    public function getWorkflows(int $page = 1, int $limit = 10)
+    {
+
+        $response =   $this->get('workflows'. '?' . http_build_query(['page' => $page, 'limit' => $limit]))['data'];
```

Comment:
there is a lot of space after `=`. Please format properly. 

This applies to all the methods in this trait.

Reply:
I ran `composer format` but it made fixes to almost every file. Do you want me to do for all the files or just the ones that I have made changes in?

Reply:
composer format uses a set of defaults that is not in sync with this repo. So I advise you don't run that on this project. 

Please it should simply be 2 spaces. @2002Bishwajeet